lib/components/tiptap/attachments/attachmentUpload.ts

```diff
@@ -134,9 +134,12 @@ export const UploadAttachment = Node.create<UploadAttachmentOptions>({
         (file: File) =>
         ({ tr, dispatch }: { tr: any; dispatch: any }) => {
           if (!dispatch || !this.options.uploadFn) return false
-
+          const MAX_FILE_SIZE = 100 * 1024 * 1024
```

Comment:
It might not be a good idea for reusability to hardcode this into Tapwrite itself.